<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>list</class>
 <widget class="QDialog" name="list">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>316</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>371</width>
     <height>241</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>百度</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>阿里</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>腾讯</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="addpushbutton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deletepushButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="companylineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteAllpushButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>全部删除</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
